src/Service/DashboardHelper.php

```diff
@@ -80,11 +80,20 @@ private function getUserColumnCaseInformation(Municipality $municipality, User $
             'count' => $this->caseRepository->findCountOfCasesWithSomeExceededDeadlineBy(['municipality' => $municipality, 'assignedTo' => $user]),
         ];
 
-        $count = array_sum(array_column($row, 'count'));
+        // TODO: When all rows are fully implemented update beneath
+//        $count = array_sum(array_column($row, 'count'));
```

Comment:
Remove this line (and maybe the “TODO” above).

Reply:
Reverted back to before in c22aeb68fb47a45b67c0be2f7a44b145efa208dc and hid unimplemented rows in dashboard grid instead.